src/chief.t.sol

```diff
@@ -207,11 +223,13 @@ contract DSChiefTest is DSThing, DSTest {
         chief.vote(slate);
         chief.lift(nonzero_slate);
         assertEq(chief.hat(), nonzero_slate);
+
+        assertTrue(!try_launch()); // chief.launch() can't be launched when hat is nonzero
```

Comment:
Doesn't this test now become more & less the same than the previous one? I think we could just leave one of them.

Reply:
I was thinking the same thing, but didn't want to delete one of Brian's tests. But I'll delete one of them now because I agree they are the same with this new addition.